docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+| \*.timings-supplement.tgz | Collected for different processing steps during the Sanger variant caller          | variant_calling_supplement | N/A                         | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+## Variant Calls
+
+### Raw SNV Calls
+
+Data files containing Single Nucleotide Variations (SNV) called from aligned reads, which have not yet been annotated but do have some filtering flags added.
+
+#### File Types
+
+| Filename Pattern | Description     | Analysis Type   | Data Category               | Generating Workflow(s)                                 |
+| ---------------- | --------------- | --------------- | --------------------------- | ------------------------------------------------------ |
+| \*.snv.vcf.gz    | VCF File format | variant_calling | Simple Nucleotide Variation | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+### Raw SV Calls
+
+Data files containing the structural variation data called from aligned reads, which have not yet been annotated or filtered.
+
+#### File Types
+
+| Filename Pattern | Description     | Analysis Type   | Data Category        | Generating Workflow(s)     |
+| ---------------- | --------------- | --------------- | -------------------- | -------------------------- |
+| \*.sv.vcf.gz     | VCF File format | variant_calling | Structural Variation | Sanger WGS Variant Calling |
+
+### Raw CNV Calls
+
+Data files containing the copy number segments data called from aligned reads, which which have not yet been annotated or filtered.
+
+#### File Types
+
+| Filename Pattern | Description     | Analysis Type   | Data Category         | Generating Workflow(s)     |
+| ---------------- | --------------- | --------------- | --------------------- | -------------------------- |
+| \*.cnv.vcf.gz    | VCF file format | variant_calling | Copy Number Variation | Sanger WGS Variant Calling |
+
+### Raw InDel Calls
+
+Data files containing the simple insertions and deletions (indels) data called from aligned reads, which have not yet been annotated but do have some filtering flags added.
+
+#### File Types
+
+| Filename Pattern | Description     | Analysis Type   | Data Category               | Generating Workflow(s)                                 |
+| ---------------- | --------------- | --------------- | --------------------------- | ------------------------------------------------------ |
+| \*.indel.vcf.gz  | VCF file format | variant_calling | Simple Nucleotide Variation | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+### VCF Index
+
+Secondary files that are external index files for VCF format files. TBI files follow a naming convention to correspond to their matching VCF file of the original filename suffixed with `.vcf.gz.tbi`. Index files are required for selective access to genomic data inside a VCF file.
```

Comment:
not sure if this page is done yet...just see some other metrics file on the diagrams on https://docs.icgc-argo.org/docs/analysis-workflows/dna-pipeline 

like Timing metrics supplement, Normal alignment QC etc,  will those be added?

Reply:
1) is in variant calling supplement 2)  is Alignment QC (coud be for normal or otherwise)